jdi-light-angular/src/main/java/com/epam/jdi/light/angular/elements/complex/ButtonToggle.java

```diff
@@ -5,26 +5,56 @@
 import com.epam.jdi.light.elements.base.UIBaseElement;
 import com.epam.jdi.light.elements.common.UIElement;
 import com.epam.jdi.light.elements.complex.WebList;
+import org.openqa.selenium.By;
 
 public class ButtonToggle extends UIBaseElement<ButtonToggleAssert> {
 
     @JDIAction("Click '{name}' button by tag value '{0}'")
-    public void clickButtonByValue(String value){
-        getButtonByTagValue(value).click();
+    public void clickButtonToggleByValue(String value) {
+        getButtonToggleByTagValue(value).click();
+    }
+
+    @JDIAction("Is '{name}' pressed '{0}'")
+    public boolean isButtonToggleButtonPressed(String value) {
```

Comment:
Assertion method name is not reasonable. From my point of view, the "isPressed" will be fine here.